docs/essentials/viewport.mdx

```diff
@@ -138,45 +136,57 @@ Update your story through [parameters](../writing-stories/parameters.mdx) to inc
 
 If you need, you can edit them on the shortcuts page.
 
-## API
+## Setting value
 
-### Parameters
+You can set the viewport in the toolbar by clicking on the viewport you want to use.
 
-This addon contributes the following [parameters](../writing-stories/parameters.mdx) to Storybook, under the `viewport` namespace:
+If you want to enforce a story to use a viewport, you can do so by setting the `viewport` global in the story:
 
-#### `defaultOrientation`
+```ts filename="Button.stories.ts|tsx" renderer="common" language="ts-4-9"
+// Replace your-framework with the name of your framework
+import type { Meta, StoryObj } from '@storybook/your-framework';
 
-Type: `'portrait' | 'landscape'`
+import { Button } from './Button';
 
-Default: `'portrait'`
+const meta = {
+  component: Button,
+} satisfies Meta<typeof Button>;
 
-Specifies the default orientation used when viewing a story.
+export default meta;
+type Story = StoryObj<typeof meta>;
 
-#### `defaultViewport`
+export const Large: Story = {
+  global: {
+    viewport: { value: 'phone', isRotated: false },
+  },
+};
+```
 
-Type: `string`
+## API
 
-Specifies the default viewport used when viewing a story. Must match a key in the [`viewports`](#viewports) object.
+### Parameters
 
-#### `disable`
+This addon contributes the following [parameters](../writing-stories/parameters.mdx) to Storybook, under the `viewport` namespace:
+
+#### `disabled`
 
 Type: `boolean`
 
 Disable this addon's behavior. If you wish to disable this addon for the entire Storybook, you should do so when registering `addon-essentials`. See the [essential addon's docs](../essentials/index.mdx#disabling-addons) for more information.
 
 This parameter is most useful to allow overriding at more specific levels. For example, if this parameter is set to `true` at the project level, it could then be re-enabled by setting it to `false` at the meta (component) or story level.
 
-#### `viewports`
+#### `options`
 
 Type:
 
 ```ts
 {
-  [key: string]: {
+  Record<string, {
     name: string,
     styles: { height: string, width: string },
-    type: 'desktop' | 'mobile' | 'tablet',
-  },
+    type: 'desktop' | 'mobile' | 'tablet' | 'other',
```

Comment:
What does 'other' refer to?

Reply:
I don't know. but it was already there in the types.
Seemed correct to just repeat here what the types had.

Reply:
It possibly is a value to indicate "don't display an icon at all"?